api/public/cerbos/response/v1/response.proto

```diff
@@ -444,8 +444,29 @@ message InspectPoliciesResponse {
   option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
     json_schema: {description: "Inspect policies response"}
   };
+
+  message Variable {
+    message Source {
+      enum Type {
+        TYPE_UNSPECIFIED = 0;
+        TYPE_EXPORTED = 1;
+        TYPE_IMPORTED = 2;
+        TYPE_LOCAL = 3;
+        TYPE_LOCAL_OR_IMPORTED = 4;
```

Comment:
Is this for variables that are defined more than once?

Reply:
`Unknown` would be better.

Reply:
```proto
enum Kind {
  KIND_UNSPECIFIED = 0;
  KIND_EXPORTED = 1;
  KIND_IMPORTED = 2;
  KIND_LOCAL = 3;
}
```

Since `KIND_UNSPECIFIED` is mandatory, I assume you mean adding `UNKNOWN` as number 4 to keep it explicit?

Reply:
Yes. `UNSPECIFIED` is a protobuf convention to indicate an unset enum so we shouldn't use it for anything else.